src/ln/channel.rs

```diff
@@ -1844,6 +1844,15 @@ impl Channel {
 		self.channel_update_count
 	}
 
+	pub fn get_channel_id(&self) -> Uint256 {
+		self.channel_id
+	}
+
+	#[test]
+	pub fn get_channel_state(&self) -> u32 {
```

Comment:
Hmm, I'd prefer this not be exposed, especially given we may want to change the enum values at some point without having to recall that the tests use it. It should be easy to instead test for an outbound Shutdown message having been generated to see the close occurring instead of getting the channel state explicitly.